test/integration/basic_test.go

```diff
@@ -181,11 +179,19 @@ func FeatureContext(s *godog.Suite) {
 }
 
 func TestMain(m *testing.M) {
+	flag.StringVar(&givenArgs, "minishift-args", "", "Arguments to pass to minishift")
+	flag.StringVar(&givenPath, "binary", "", "Path to minishift binary")
+	flag.StringVar(&givenTags, "tags", "", "Tags of tests to pass to godog")
+	flag.Parse()
+
+	givenTags = strings.Replace(givenTags, " ", ", ", -1)
```

Comment:
A couple of things, I think this should be guarded with an if statement, if no tags are given the 'tag' argument should not even be set in the good.Options.

Secondly, why do we not pass the tags comma separated in the first place?